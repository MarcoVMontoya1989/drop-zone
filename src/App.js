import './App.css';
import './styles/Base.scss';
import 'bulma/css/bulma.min.css';
import HomePageComponent from "./Components/Pages/HomePage.component";
import {Fragment} from "react";
import NavbarComponent from "./Components/Component/Navbar.component";

function App() {
  return (
    <Fragment>
      <section className="hero is-fullheight is-default">
        <div className="hero-head head-mobile">
          <NavbarComponent/>
        </div>

        <div className="hero-body">
          <HomePageComponent/>
        </div>
      </section>
    </Fragment>
  );
}

export default App;
