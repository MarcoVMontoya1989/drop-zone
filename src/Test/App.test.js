import { render } from '@testing-library/react';
import {shallow} from 'enzyme';

import App from '../App';
import NavbarComponent from "../Components/Component/Navbar.component";
import HomePageComponent from "../Components/Pages/HomePage.component";

const wrapper = shallow(<App />)

describe('', function () {
  test('Should render App component', () => {
    render(<App />);
  });

  test('It should render component', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('It should render child component Navbar', () => {
    expect(wrapper.containsMatchingElement(<NavbarComponent />)).toEqual(true);
  });

  test('It should render child component HomePageComponent', () => {
    expect(wrapper.containsMatchingElement(<HomePageComponent />)).toEqual(true);
  });

});