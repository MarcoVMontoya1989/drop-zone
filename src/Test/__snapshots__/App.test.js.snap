// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` It should render component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="hero is-fullheight is-default"
      >
        <div
          class="hero-head head-mobile"
        >
          <nav
            class="navbar"
          >
            <div
              class="container"
            >
              <div
                class="navbar-end navbar-end-force"
              >
                <a
                  class="navbar-item navbar-item-drop"
                >
                  <div
                    class="columns drop-zone"
                  >
                    <div
                      class="column drop-content"
                      id="rohza-text"
                    >
                      Drop
                    </div>
                    <div
                      class="column drop-content drop-content-end"
                      id="rohza-text"
                    >
                      Zone
                    </div>
                  </div>
                </a>
              </div>
            </div>
          </nav>
        </div>
        <div
          class="hero-body"
        >
          <div
            class="container has-text-centered"
          >
            <div
              class="columns is-vcentered"
            >
              <div
                class="column is-5"
              />
              <div
                class="column is-6 is-offset-1"
              >
                <h1
                  class="title is-2"
                  id="title-roboto"
                >
                  LEEZY BOOST 103
                </h1>
                <div
                  class="container clock-container"
                >
                  <div
                    class="clock-column"
                  >
                    <div
                      class="clock-count"
                    >
                      <p
                        class="clock-day clock-timer"
                      >
                        1
                      </p>
                      <p
                        class="clock-day clock-timer"
                      >
                        3
                      </p>
                    </div>
                    <p
                      class="clock-label"
                    >
                      Days
                    </p>
                  </div>
                  <div
                    class="clock-column"
                  >
                    <div
                      class="clock-count"
                    >
                      <p
                        class="clock-hours clock-timer"
                      >
                        0
                      </p>
                      <p
                        class="clock-hours clock-timer"
                      >
                        8
                      </p>
                    </div>
                    <p
                      class="clock-label"
                    >
                      Hours
                    </p>
                  </div>
                  <div
                    class="clock-column"
                  >
                    <div
                      class="clock-count"
                    >
                      <p
                        class="clock-minutes clock-timer"
                      >
                        1
                      </p>
                      <p
                        class="clock-minutes clock-timer"
                      >
                        2
                      </p>
                    </div>
                    <p
                      class="clock-label"
                    >
                      Mins
                    </p>
                  </div>
                  <div
                    class="clock-column"
                  >
                    <div
                      class="clock-count"
                    >
                      <p
                        class="clock-seconds clock-timer"
                      >
                        0
                      </p>
                      <p
                        class="clock-seconds clock-timer"
                      >
                        5
                      </p>
                    </div>
                    <p
                      class="clock-label"
                    >
                      Secs
                    </p>
                  </div>
                </div>
                <p
                  id="bottom-title-roboto"
                >
                  UNTIL DROP
                </p>
              </div>
            </div>
          </div>
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="hero is-fullheight is-default"
    >
      <div
        class="hero-head head-mobile"
      >
        <nav
          class="navbar"
        >
          <div
            class="container"
          >
            <div
              class="navbar-end navbar-end-force"
            >
              <a
                class="navbar-item navbar-item-drop"
              >
                <div
                  class="columns drop-zone"
                >
                  <div
                    class="column drop-content"
                    id="rohza-text"
                  >
                    Drop
                  </div>
                  <div
                    class="column drop-content drop-content-end"
                    id="rohza-text"
                  >
                    Zone
                  </div>
                </div>
              </a>
            </div>
          </div>
        </nav>
      </div>
      <div
        class="hero-body"
      >
        <div
          class="container has-text-centered"
        >
          <div
            class="columns is-vcentered"
          >
            <div
              class="column is-5"
            />
            <div
              class="column is-6 is-offset-1"
            >
              <h1
                class="title is-2"
                id="title-roboto"
              >
                LEEZY BOOST 103
              </h1>
              <div
                class="container clock-container"
              >
                <div
                  class="clock-column"
                >
                  <div
                    class="clock-count"
                  >
                    <p
                      class="clock-day clock-timer"
                    >
                      1
                    </p>
                    <p
                      class="clock-day clock-timer"
                    >
                      3
                    </p>
                  </div>
                  <p
                    class="clock-label"
                  >
                    Days
                  </p>
                </div>
                <div
                  class="clock-column"
                >
                  <div
                    class="clock-count"
                  >
                    <p
                      class="clock-hours clock-timer"
                    >
                      0
                    </p>
                    <p
                      class="clock-hours clock-timer"
                    >
                      8
                    </p>
                  </div>
                  <p
                    class="clock-label"
                  >
                    Hours
                  </p>
                </div>
                <div
                  class="clock-column"
                >
                  <div
                    class="clock-count"
                  >
                    <p
                      class="clock-minutes clock-timer"
                    >
                      1
                    </p>
                    <p
                      class="clock-minutes clock-timer"
                    >
                      2
                    </p>
                  </div>
                  <p
                    class="clock-label"
                  >
                    Mins
                  </p>
                </div>
                <div
                  class="clock-column"
                >
                  <div
                    class="clock-count"
                  >
                    <p
                      class="clock-seconds clock-timer"
                    >
                      0
                    </p>
                    <p
                      class="clock-seconds clock-timer"
                    >
                      5
                    </p>
                  </div>
                  <p
                    class="clock-label"
                  >
                    Secs
                  </p>
                </div>
              </div>
              <p
                id="bottom-title-roboto"
              >
                UNTIL DROP
              </p>
            </div>
          </div>
        </div>
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
